components/data/buckets.ts

```diff
@@ -0,0 +1,94 @@
+import type React from 'react';
+
+import IconGettingStarted from '../icons/GettingStarted';
+import IconGuide from '../icons/Guide';
+import IconMigration from '../icons/Migration';
+import IconSpec from '../icons/Spec';
+import IconTutorials from '../icons/Tutorials';
+import IconUseCases from '../icons/UseCases';
+import IconUsers from '../icons/Users';
+
+type IconType = (props: any) => React.ReactElement;
```

Comment:
Kindly define this type in `types/buttons` folder. This type has been used multiple times at other places also.

Reply:
Is defining it in the `types/buttons` folder accurate? There is a folder `types/components/buttons` where I can define it. But I think it will be more accurate to define it in a separate file `IconType.ts` in the folder `types/components` as it is used in multiple places. What's your suggestion on this? For now, I have defined it in separate file, but let me no if you want me to place it somewhere else, I will move it.